### PR TITLE
Add spec helper to simplify specs, and Gemfile to load required dev gems

### PR DESCRIPTION
**Pull request by jemmyw**

### CODE REVIEW

**Review by percyhanna — APPROVED** [review]
Looks good to me, but maybe another set of eyes would be good. 👍 

Could we add CircleCI integration as well?

**Comment by jemmyw** [comment]
@percyhanna ok for me to just merge these spec only changes? We should add circle in another PR I think

**Comment by percyhanna** [comment]
@jemmyw Go for it, that can be iterated on after. 👍

File: .rspec [hunk 1]
```diff
@@ -1 +1,3 @@
---color
\ No newline at end of file
+--colour
```
Comment (avit) [inline .rspec]: 🇨🇦 🇬🇧 🇳🇿

File: Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,60 @@
+PATH
```
Comment (avit) [inline Gemfile.lock]: I don't think lockfiles should be included in gems: could we ignore this? They should be testable with whatever is the latest.

File: html2confluence.gemspec [hunk 3]
```diff
@@ -15,4 +15,7 @@ Gem::Specification.new do |s|
   s.files        = Dir.glob("{lib,spec}/**/*") + %w(example.rb README.mdown)
 
   s.add_dependency "nokogiri"
+  s.add_development_dependency "rspec"
+  s.add_development_dependency "rubocop", "< 0.56"
```
Comment (avit) [inline html2confluence.gemspec]: Why does this need to be pinned down?

**Comment by schneidmaster** [comment]
Superseded by #18

**Pull request closed without merging**